src/main/java/com/mraof/minestuck/data/recipe/MinestuckRecipeProvider.java

```diff
@@ -312,7 +330,6 @@ protected void buildRecipes(Consumer<FinishedRecipe> recipeBuilder)
 		SingleItemRecipeBuilder.stonecutting(Ingredient.of(MSBlocks.COARSE_STONE.get()), RecipeCategory.BUILDING_BLOCKS, MSBlocks.COARSE_STONE_STAIRS.get()).unlockedBy("has_coarse_stone", has(MSBlocks.COARSE_STONE.get())).save(recipeBuilder, new ResourceLocation(Minestuck.MOD_ID, "coarse_stone_stairs_from_coarse_stone_stonecutting"));
 		SingleItemRecipeBuilder.stonecutting(Ingredient.of(MSBlocks.COARSE_STONE.get()), RecipeCategory.BUILDING_BLOCKS, MSBlocks.COARSE_STONE_SLAB.get(), 2).unlockedBy("has_coarse_stone", has(MSBlocks.COARSE_STONE.get())).save(recipeBuilder, new ResourceLocation(Minestuck.MOD_ID, "coarse_stone_slab_from_coarse_stone_stonecutting"));
 		SingleItemRecipeBuilder.stonecutting(Ingredient.of(MSBlocks.COARSE_STONE_BRICKS.get()), RecipeCategory.BUILDING_BLOCKS, MSBlocks.COARSE_STONE_BRICK_STAIRS.get()).unlockedBy("has_coarse_stone_bricks", has(MSBlocks.COARSE_STONE_BRICKS.get())).save(recipeBuilder, new ResourceLocation(Minestuck.MOD_ID, "coarse_stone_brick_stairs_from_coarse_stone_bricks_stonecutting"));
-		SingleItemRecipeBuilder.stonecutting(Ingredient.of(MSBlocks.COARSE_STONE_BRICKS.get()), RecipeCategory.BUILDING_BLOCKS, MSBlocks.COARSE_STONE_BRICK_SLAB.get(), 2).unlockedBy("has_coarse_stone_bricks", has(MSBlocks.COARSE_STONE_BRICKS.get())).save(recipeBuilder, new ResourceLocation(Minestuck.MOD_ID, "coarse_stone_brick_slab_from_coarse_stone_bricks_stonecutting"));
```

Comment:
Looks like this recipe was removed without a replacement. Was that intentional? Same thing for the frost tile slab and frost brick slab.

Reply:
Not intentional. iirc, I accidentally duplicated those lines, must have gotten lost in trying to remove the dups. Added all three slabs back.